gpg-interface.c

```diff
@@ -394,6 +497,14 @@ int git_gpg_config(const char *var, const char *value, void *cb)
 		return 0;
 	}
 
+	if (!strcmp(var, "gpg.ssh.allowed-signers")) {
```

Comment:
Typically, Git's config keys are camelCased (and downcased in the `*_config()` functions, i.e. you would need to change this to `gpg.ssh.allowedsigners`, same with `gpg.ssh.revocationfile` below).

Reply:
I've fixed the style changes and the config keys.
I'm looking for more feedback on how to actually do the configuration of the ssh public keys (or private keyfiles in case no agent is used). I'm going to add docs and tests then before actually submitting the patch.